portfolio/src/main/webapp/style.css

```diff
@@ -126,4 +131,13 @@ html, body, section {
     height: 100%;
     margin: 0px;
     text-align: center;
+}
+
+section {
+    padding: 10px;
+}
```

Comment:
small nit: add a new line in between these.

In general, you want to try to keep your formatting consistent with the rest of the file. There are also a few areas where you indent with tabs and others where you indent with spaces. I'd recommend choosing one or the other to keep things consistent.

Reply:
Thanks for the feedback, I'll do that now!